frontend/fingertips-frontend/components/forms/SearchForm/__snapshots__/SearchForm.test.tsx.snap

```diff
@@ -2,6 +2,114 @@
 
 exports[`snapshot test - renders the form 1`] = `
 <DocumentFragment>
+  .c0 {
+  color: #0b0c0c;
+  font-family: "nta",Arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 1.09375;
+  display: block;
+  margin-top: 0;
+  margin-bottom: 30px;
+}
+
+@media print {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media print {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media print {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media print {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media print {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
+
+@media only screen and (min-width: 641px) {
+
+}
```

Comment:
Not sure why we are getting these additional empty css classes added to our snapshot when all that was added was a H1 tag to the search form - Was something else changed?

Reply:
Yeah just adding the H1 tag caused all this 😢  - It's a shame but I think we just have to live with it for now and keep an eye on it. I think we need to use the snapshot tests very sparingly going forward.